saleor/wishlist/models.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import unicode_literals
+
+from uuid import uuid4
+
+from django.contrib.postgres.fields import HStoreField
+from django.db import models
+from django.core.urlresolvers import reverse
+from django.utils.encoding import python_2_unicode_compatible
+
+from saleor.core.utils import build_absolute_uri
+from ..product.models import ProductVariant, Product
+from ..userprofile.models import User
+
+
+@python_2_unicode_compatible
+class Wishlist(models.Model):
+    user = models.ForeignKey(User)
+    token = models.CharField(default=uuid4, editable=False, max_length=36,
+                             unique=True)
+    public = models.BooleanField(default=False)
+
+    def change_token(self):
+        self.token = uuid4()
+        self.save(update_fields=['token'])
+
+    def __str__(self):
+        if self.company_name:
+            return '%s - %s' % (self.company_name, self.full_name)
+        return self.full_name
+
+    def __repr__(self):
```

Comment:
You don't have to override default `__repr__` in django models, default implementation will execute `str(self)`